interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is the code thats on your device that you can work on locally. Github is a free online resource kind of like an internet for coding where many people can work on the same codes, modify them, and retrieve them. One is for coding on your local machine, and one is a hub for codes. 
 
-Researched answer:
+Researched answer: Git is in your local system, not the cloud. Github IS esxclusively cloud-based. As long as you have the code you could use git offline. They both track changes. Git is a software command-line tool and github is a service, a graphical user interface. Git came first in 2005 and Github in 2008.  
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: a strict equality will return a boolean value. If the values match in data type and the data itself, then it will return 'true'. If they are different in any way like data type for example, 3 and three, it will come back false. 
 
-Researched answer:
+Researched answer: Both loose and strict equality operators return a boolean value. loose equality will convert both data types then compare them (pretty cool) but the strict equality does not convert any of the data, the data has to be exctly the same value and data type or it will be false.
```

Comment:
There are also a few other types of operators that will return a Boolean value.  Relational and Logical operators.